scripts/aem.js

```diff
@@ -355,43 +355,7 @@ function decorateTemplateAndTheme() {
   if (theme) addClasses(document.body, theme);
 }
 
-/**
```

Comment:
do we have to delete this? can't we just overlay and use the function from scripts.js?

Reply:
I asked David about this yesterday. He said aem.js belongs to the customer project, it is open source and we won't be pushing any kind of updates to the file. 
To leave it as ‘clean’ as possible though, I moved the buttons since I did a lot of customization to them and left a placeholder comment for anyone who looks for them there.